tooling/gulpfile.js

```diff
@@ -28,27 +28,26 @@ const rootFolder           = path.join(__dirname, '../');
 
 // General configurations for the build process
 const config = {
-  libName: libName,
+  libName,
   paths: {
-    rootFolder: rootFolder,
+    rootFolder,
     distFolder: path.join(rootFolder, 'dist/terminus-ui'),
     npmFolder: path.join(rootFolder, 'node_modules/@terminus/ui'),
     scss: {
       helpersInputs: [
         path.join(rootFolder, 'terminus-ui', 'scss/helpers/_a11y.scss'),
-        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_typography.scss'),
-        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_cursors.scss'),
-        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_color.scss'),
+        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_animation.scss'),
         path.join(rootFolder, 'terminus-ui', 'scss/helpers/_assets.scss'),
         path.join(rootFolder, 'terminus-ui', 'scss/helpers/_breakpoints.scss'),
+        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_color.scss'),
+        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_cursors.scss'),
         path.join(rootFolder, 'terminus-ui', 'scss/helpers/_layout.scss'),
```

Comment:
input-placeholder
layout
menu
opposite-direction

Reply:
We only expose helpers that our consumers should be using - not every helper we have. For example, if a consumer is using `input-placeholder` that means they are not using our input.

What helpers we expect or allow our consumers to use may change over time, but I believe this set represent the current expectation.